doc-content/enterprise-only/openshift/template-overview-con.adoc

```diff
@@ -5,46 +5,46 @@
 
-* `rhpam72-kieserver-postgresql.yaml` provides a {KIE_SERVER} and a PostgreSQL instance that the {KIE_SERVER} uses. You can configure the {KIE_SERVER} to connect to a {CENTRAL}. Also, you can copy sections from this template into another template to configure a {KIE_SERVER} in the other template to use PostgreSQL and to provide the PostgreSQL instance. For details about this template, see <<rhpam72-kieserver-mysql-ref>>.
+* `rhpam73-kieserver-postgresql.yaml` provides a {KIE_SERVER} and a PostgreSQL instance that the {KIE_SERVER} uses. You can configure the {KIE_SERVER} to connect to a {CENTRAL}. Also, you can copy sections from this template into another template to configure a {KIE_SERVER} in the other template to use PostgreSQL and to provide the PostgreSQL instance. For details about this template, see <<rhpam73-kieserver-mysql-ref>>.
 endif::[]
 
 ifeval::["{context}"=="openshift-trial"]
-* `rhpam72-trial-ephemeral.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. This environment uses an ephemeral configuration without any persistent storage. For details about this template, see <<rhpam72-trial-ephemeral-ref>>.
+* `rhpam73-trial-ephemeral.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. This environment uses an ephemeral configuration without any persistent storage. For details about this template, see <<rhpam73-trial-ephemeral-ref>>.
 endif::[]
 endif::PAM[]
 ifdef::DM[]
 ifeval::["{context}"=="openshift-authoring-managed"]
-* `rhdm72-authoring.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. You can use this environment to author services and other business assets or to run them in staging or production environments. For details about this template, see <<rhdm72-authoring-ref>>.
+* `rhdm73-authoring.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. You can use this environment to author services and other business assets or to run them in staging or production environments. For details about this template, see <<rhdm73-authoring-ref>>.
 
-* `rhdm72-authoring-ha.yaml` provides a high-availability {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. You can use this environment to author services and other business assets or to run them in staging or production environments. The high-availability functionality is in technical preview. For details about this template, see <<rhdm72-authoring-ha-ref>>.
+* `rhdm73-authoring-ha.yaml` provides a high-availability {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. You can use this environment to author services and other business assets or to run them in staging or production environments. The high-availability functionality is in technical preview. For details about this template, see <<rhdm73-authoring-ha-ref>>.
```

Comment:
Minor suggestion - maybe changing this to a note to highlight that this is technology preview:

```
Note
Configuration of {CENTRAL} for high availability is currently technology preview.
```